x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/index.tsx

```diff
@@ -84,6 +90,12 @@ MyLabelButton.defaultProps = {
   flush: 'right',
 };
 
+const RuleTypeEuiFormRow = styled(EuiFormRow).attrs<{ $isVisible: boolean }>(({ $isVisible }) => ({
```

Comment:
What is the purpose of `$` at the begging of the variable? Just curious :)

Reply:
https://styled-components.com/docs/api#transient-props